oShare/AppDelegate.swift

```diff
@@ -1,11 +1,3 @@
-//
-//  AppDelegate.swift
-//  oShare
-//
-//  Created by Ben Deckys on 24/12/17.
-//  Copyright © 2017 MIMIMI. All rights reserved.
-//
-
```

Comment:
I have developed the habit of removing these lines in files belonging specifically to my project. If a file belongs to a third party, and is brought in to the codebase via standard "drag and drop", I leave the headerdoc in to disambiguate what comes from where.